Source/WebKit/Shared/Cocoa/AuxiliaryProcessCocoa.mm

```diff
@@ -321,5 +322,17 @@ static void initializeTimerCoalescingPolicy()
     return isSystemWebKit;
 }
 
+void AuxiliaryProcess::setNotifyOptions()
+{
+#if !ENABLE(NOTIFY_BLOCKING)
+    uint32_t opts = 0;
+#if ENABLE(NOTIFY_FILTERING)
+    opts |= NOTIFY_OPT_DISPATCH | NOTIFY_OPT_REGEN | NOTIFY_OPT_FILTERED;
+#endif
+    if (!opts)
+        return;
+    notify_set_options(opts);
+#endif
```

Comment:
It looks like this can be simplified as:
```
#if !ENABLE(NOTIFY_BLOCKING) && ENABLE(NOTIFY_FILTERING)
    notify_set_options(NOTIFY_OPT_DISPATCH | NOTIFY_OPT_REGEN | NOTIFY_OPT_FILTERED)
#endif
```

Reply:
Ah, nice catch, fixed in latest patch!

Thanks for reviewing!